docs/server/production-deploy.md

```diff
@@ -161,16 +161,6 @@ You may sometimes want to have multiple parallel deployments on the same cluster
 - when you want to split Temporal deployments based on namespaces, e.g. staging/dev/uat, or for different teams who need to share common infrastructure.
 - when you need a new deployment to change `numHistoryShards`.
 
-**We recommend not doing this if you can avoid it**. If you need to do it anyway, double-check the following:
```

Comment:
We still need this for per-1.19 servre. I think we should keep this doc with note that if you are running v1.19+, you don't need to do that.

Reply:
Updated